### PR TITLE
adding RBAC test to e2e tests

### PR DESCRIPTION
**Pull request by jaidevmane**
Signed-off-by: Jai Subash Devmane <jdevmane@redhat.com>

Thank you for contributing to Velero!

# Please add a summary of your change
- Added an E2E test that verifies service accounts, cluster roles, and cluster rolebindings are backed up and restored correctly.

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by jaidevmane** [comment]
All checks passed. Ready to review.

File: test/e2e/common.go [hunk 1]
```diff
@@ -47,10 +48,85 @@ func createNamespace(ctx context.Context, client testClient, namespace string) e
 	return err
 }
 
+// createNamespaceAndRbac creates a kubernetes namespace, service account, clusterrole, and clusterrolebinding
```
Comment (dsu-igeek) [inline test/e2e/common.go:51]: Let's break this into separate functions.  This is not very re-usable, everything needs to be done together.

**Review by dsu-igeek — CHANGES_REQUESTED** [review]
Looks like this needs to be rebased.  Would you split up the createNamespaceAndRbac into some smaller functions - it seems like this is only useful for this case, so it's not very common.

**Comment by ywk253100** [comment]
Similar to https://github.com/vmware-tanzu/velero/pull/3944, let's expand the existing `Basic` test case to include the RBAC resources.
Both of these two PRs need some adjustment of the `Basic` test case before expanding it, see https://github.com/vmware-tanzu/velero/pull/3944#issuecomment-909818616

**Comment by dsu-igeek** [comment]
Leaving this open until #4099 is addressed

**Comment by qiuming-best** [comment]
I'll keep going on the works on #4455, so I closed

**Pull request closed without merging**